extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -108,6 +127,75 @@ const renderDefaultPageAttachmentUI = (pageEl, attachmentEl) => {
   return openAttachmentEl;
 };
 
+/**
+ * Renders inline page attachment UI.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @return {!Element}
+ */
+ const renderOutlinkPageAttachmentUI = (win, pageEl, attachmentEl, attachmentHref) => {
+  const openAttachmentEl = buildOpenOutlinkAttachmentElement(pageEl);
+
+  // Copy href to the element so it can be previewed on hover and long press.
+  if (attachmentHref) {
+    openAttachmentEl.setAttribute('href', attachmentHref);
+  }
+  
+  openAttachmentEl.addEventListener('click', () =>
+    openAttachmentEl.openAttachment()
+  );
```

Comment:
I might be missing sth, but isn't this already a `<a>` (so the click on it will navigate)?

Reply:
actually yeah it's fine without it. before it was blocking for page navigation for some reason but I guess something fixed that issue